webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -0,0 +1,260 @@
+import React from 'react';
+import {GitPullRequestIcon, IssueOpenedIcon, IconProps} from '@primer/octicons-react';
+import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+import {Badge, Tooltip, OverlayTrigger} from "react-bootstrap";
+import * as CSS from 'csstype';
+import CrossIcon from "../../images/icons/cross";
+import DotIcon from "../../images/icons/dot";
+import TickIcon from "../../images/icons/tick";
+import SignIcon from '../../images/icons/sign';
+import {formatTimeSince} from '../../utils/date_utils';
+import {GitlabItemsProps, Label} from "../../types/gitlab_items"
+
+export const notificationReasons: Record<string, string> = {
```

Comment:
@mickmister what is the process to define/validate the wording for the plugin?

Reply:
@cwarnermm Would you be able to take a look at the user-facing strings here?